astropy/table/table_helpers.py

```diff
@@ -137,11 +137,27 @@ def complex_table():
     return table
 
 
+class ArrayWrapperInfo(ParentDtypeInfo):
```

Comment:
This is basically the same as `NdarrayMixinInfo` - can we use just one of the two?

Reply:
`ArrayWrapper` was made as a light example of a mixin column and in fact doesn't quite work with all our machinery, so I'm happy to remove it. Let's plan that for 5.0 though.

Reply:
OK, sounds good. Maybe add a comment to that extent? (or raise an issue)